src/components/NewLocationPage/SummaryStat/StatChart.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Metric } from 'common/metricEnum';
+import { getSparkLineProps, MetricToSparkLine } from '../SparkLineBlock/utils';
+import { Projection } from 'common/models/Projection';
+import SparkLine from '../SparkLineBlock/SparkLine';
+import VaccinationProgressBarBlock from '../VaccinationProgressBarBlock';
+
+const StatChart: React.FC<{ projection: Projection; metric: Metric }> = ({
+  metric,
+  projection,
+}) => {
+  const sparkLineMetric = MetricToSparkLine[metric];
+  const sparkLineProps = getSparkLineProps(sparkLineMetric, projection);
+
+  if (metric === Metric.VACCINATIONS) {
+    return (
+      <VaccinationProgressBarBlock
+        projection={projection}
+        locationName={'New York'}
```

Comment:
I don't think this should always be "New York". 
![image](https://user-images.githubusercontent.com/46338131/167464008-f9c76863-41c0-4e31-ba22-e394f59ba39a.png)

Reply:
oh whoops good catch